### PR TITLE
fix errors in elvToken abigen go binding

### PR DESCRIPTION
**Pull request by elv-preethi**
In contracts-go, the `elv_tradable_full.go` and `elvtoken.go` is  in common package `tradable`  so the same global variables was present in both the files and also erc20 contract methods was also common between them. 
So, made the changes to script `build-go-payments.sh` to generate elvToken abigen in package `token` instead. 

We need to call ERC20 contract to check the `allowance` and also to call `approve` for ethereum payments.

The `tmp-ethereum-payments-tools` elv-master branch uses this contract now.

### CODE REVIEW

File: build-go-payments.sh [hunk 1]
```diff
@@ -48,7 +48,7 @@ solc_0_8_13_dir=$(realpath "$1")
     ./abigen --abi "${payments_out_dir}/ERC20Payments.abi" --bin "${payments_out_dir}/ERC20Payments.bin" --pkg payments --out "${payments_abigen_dir}/erc20_payments.go"
 
     echo "#### generating latest src/tradable/ElvToken.sol"
-    ./abigen --abi "${elv_token_out_dir}/ElvToken.abi" --bin "${elv_token_out_dir}/ElvToken.bin" --pkg tradable --out "${elv_token_abigen_dir}/ElvToken.go"
+    ./abigen --abi "${elv_token_out_dir}/ElvToken.abi" --bin "${elv_token_out_dir}/ElvToken.bin" --pkg token --out "${elv_token_abigen_dir}/ElvToken.go"
```
Comment (elv-gilles) [inline build-go-payments.sh]: `go` files are usually lower and snake case.

```suggestion
    ./abigen --abi "${elv_token_out_dir}/ElvToken.abi" --bin "${elv_token_out_dir}/ElvToken.bin" --pkg token --out "${elv_token_abigen_dir}/elv_token.go"
```

**Review by elv-gilles — APPROVED** [review]
Looks good.

Just a note (maybe a useless one) that the indexer does not look at 'elv_token' for now so the event `TransferERC20` (`0xddf252ad...f523b3ef`) is taken from `tradable`.

File: build-go-payments.sh (hunk 1 above)
Comment (elv-preethi) [reply build-go-payments.sh]: oh yes made the changes.

**Comment by elv-preethi** [comment]
> Looks good.
> 
> Just a note (maybe a useless one) that the indexer does not look at 'elv_token' for now so the event `TransferERC20` (`0xddf252ad...f523b3ef`) is taken from `tradable`.

Not sure we need this for indexer.

**Pull request merged**